systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthPlainIsolatedST.java

```diff
@@ -718,6 +727,33 @@ void testSaslPlainAuthenticationKafkaConnectIsAbleToConnectToKafkaOAuth(Extensio
         KafkaConnectUtils.waitForConnectReady(testStorage.getNamespaceName(), testStorage.getClusterName());
     }
 
+    @ParallelTest
+    void testOAuthKafkaMetrics(ExtensionContext extensionContext) {
+        TestStorage testStorage = new TestStorage(extensionContext);
+        List<String> expectedMetrics = Arrays.asList("strimzi_oauth_http_requests_maxtimems", "strimzi_oauth_http_requests_mintimems",
+            "strimzi_oauth_http_requests_avgtimems", "strimzi_oauth_http_requests_totaltimems", "strimzi_oauth_http_requests_count");
+
+        // Collect metrics and verify
+        resourceManager.createResource(extensionContext, ScraperTemplates.scraperPod(testStorage.getNamespaceName(), testStorage.getScraperName()).build());
+        String scraperName = kubeClient().listPodsByPrefixInName(testStorage.getNamespaceName(), testStorage.getScraperName()).get(0).getMetadata().getName();
+
+        MetricsCollector metricsCollector = new MetricsCollector.Builder()
+            .withNamespaceName(testStorage.getNamespaceName())
+            .withScraperPodName(scraperName)
+            .withComponentName(oauthClusterName)
+            .withComponentType(ComponentType.Kafka)
+            .build();
+
+        metricsCollector.collectMetricsFromPods();
+        Collection<String> resultMetrics = metricsCollector.getCollectedData().values();
+
+        for (String result : resultMetrics) {
+            for (String expectedMetric : expectedMetrics) {
+                assertTrue(result.contains(expectedMetric));
+            }
+        }
```

Comment:
For debug purposes, it would be nice to have some debug log with collected data

Reply:
Added logging